### PR TITLE
Internationalization and xother issue fixed

### PR DESCRIPTION
**Pull request by biodivAtlasAT**
Internationalization - dynmic language support

ALAChart.js

description
the ala-collectory /data/ala-collectory/config/charts.json

includes the variable by Family
there you can set includeOther to true or to now to false
then the URL of biocacheService includes the variable
&xother= within true or false

best regards
Georg

### CODE REVIEW

File: grails-app/assets/javascripts/ALAChart.js [hunk 1]
```diff
@@ -133,15 +133,21 @@ ALA.BiocacheCharts = function (chartsDivId, chartOptions) {
                 datastructure.datasets[keySeries].hoverBorderColor = transparentColors(segmentColor, 85);
                 datastructure.datasets[keySeries].borderWidth = 1;
             }
-
-            if (chartConfig.facet != resultSeries.label) datastructure.datasets[keySeries].label = resultSeries.label
+            //if (chartConfig.facet != resultSeries.label) datastructure.datasets[keySeries].label = resultSeries.label
```
Comment (vjrj) [inline grails-app/assets/javascripts/ALAChart.js:136]: I only have minor comments, in general LGreatTM. Can you remove this?

File: grails-app/conf/application.yml [hunk 2]
```diff
@@ -8,3 +8,8 @@ info:
         name: '@info.app.name@'
         version: '@info.app.version@'
         grailsVersion: '@info.app.grailsVersion@'
+
```
Comment (vjrj) [inline grails-app/conf/application.yml:11]: Can you remove this also?

**Review by vjrj — APPROVED** [review]
Looks great to me, thanks Georg. I tested and worked for me. If you can remove the minor things I commented, or if not we can remove it later.

File: grails-app/conf/application.yml [hunk 3]
```diff
@@ -8,3 +8,8 @@ info:
...
+environments:
+    development:
+        charts:
+            uriPrefix: D:/reinhardt/firma/Biodiversity/working
```
Comment (vjrj) [inline grails-app/conf/application.yml:15]: This can be also commented, like this only will work for reinhard....

**Review by djtfmartin — APPROVED** [review]

**Pull request merged**